frontend/lang/french.json

```
@@ -0,0 +1,337 @@
+    "t_logs_click_for_more_info": "Cliquer pour plus d'infos",
+    
+    "t_ready_notif_title": "Pluto : notebook prêt",
+    "t_ready_notif_body": "✓ Toutes les {{count}} étapes terminées",
+    "t_ready_notif": "Notifier quand terminé",
+    "t_ready_notif_permission": "Pluto a besoin de permission pour afficher les notifications. <strong>Active les notifications</strong> dans les paramètres de ton navigateur pour utiliser cette fonctionnalité.",
+    
+    "t_pkg_disables_str": "{{function}} désactive le gestionnaire de packages intégré de Pluto.",
+    "t_pkg_stdlib": "{{package}} fait partie de la <em>bibliothèque standard</em> pré-installée de Julia.",
+    "t_pkg_will_be_installed": "{{package}} sera installé quand tu exécuteras ce notebook.",
+    "t_pkg_is_installing": "{{package}} est en cours d'installation...",
+    "t_pkg_is_installed": "{{package}} est installé dans le notebook.",
+    
+    "t_pkg_not_found": "Le package {{package}} n'a pas pu être trouvé dans le registre. As-tu fait une faute de frappe ?",
+    "t_pkg_will_be_installed_in_notebook": "{{package}} sera installé dans le notebook quand tu exécuteras cette cellule.",
+    
+    "t_pkg_installation_can_take": "L'installation peut prendre {{time_install}}.<br/>Après, il se charge en {{time_load}}.",
+    "t_pkg_update_packages": "Mettre à jour les packages",
+    "t_pkg_update_packages_description": "Veux-tu vérifier les mises à jour et les installer ? Une sauvegarde du fichier notebook sera créée.",
+    "t_pkg_currently_busy": "Pkg est actuellement occupé avec d'autres packages... reviens plus tard !",
+    "t_pkg_toggle_terminal": "Afficher/masquer la sortie terminal de Pkg",
+    
+    "t_pkg_go_to_help": "En savoir plus sur les packages dans Pluto",
+    
+    "t_save_all_changes": "Sauvegarder tous les changements",
+    "t_save_all_changes_description": "Sauvegarder et exécuter toutes les cellules modifiées",
+    "t_file_change_detected": "Changement de fichier détecté, <b>notebook mis à jour</b>",
+    "t_file_saved": "Sauvegardé",
+    
+    "t_recording_ui_start_recording": "<strong>Commencer l'enregistrement</strong>",
+    "t_recording_ui_start_recording_muted": "<strong>Commencer l'enregistrement</strong> (sans audio)",
+    "t_recording_ui_stop_recording": "<strong>Arrêter l'enregistrement</strong>",
+    "t_recording_ui_back_to_recording": "Retour à <strong>l'enregistrement</strong> {{icon}}",
+    
+    "t_presentation_previous_slide": "Diapositive précédente",
+    "t_presentation_next_slide": "Diapositive suivante",
+    
+    
+    "t_status_names": {
+        "workspace": "Configuration de l'espace de travail",
+        "create_process": "Démarrage de Julia",
+        "init_process": "Initialisation",
+        "pkg": "Gestion des packages",
+        "instantiate1": "instanciation",
+        "instantiate2": "instanciation",
+        "instantiate3": "instancier",
```

```suggestion
        "instantiate3": "instanciation",
```
although this term seems very obscure in French